docs/content/docs/external_auth_opa.md

```diff
@@ -0,0 +1,119 @@
+# OPA-plugin-enabled OSM POC
+
+## Overview and limitations
+- Allows configuring an envoy's [External Authorization extension](https://www.envoyproxy.io/docs/envoy/latest/configuration/http/http_filters/ext_authz_filter) through OSM's configmap.
+- Authorization filter is currently applied in inmesh `inbound` and `ingress` connections. 
+- This demo DOES NOT inject the OPA plugin sidecar on every pod, though this seems to be the intended model for OPA to run with for obvious latency reasons.
+  
+If using `values.yaml`:
+ ```
+  # External authz
+  inbound_extauthz:
+    enable: true
+    address: opa.opa.svc.cluster.local
+    port: 9191
+    statPrefix: authz_opa
+    timeout: 1s
+    failureModeAllow: false
+```
+Example OSM's configmap deployed by this POC:
+```
+  inbound_extauthz_enable: true
+  inbound_extauthz_address: opa.opa.svc.cluster.local
+  inbound_extauthz_port: 9191
+  inbound_extauthz_statprefix: authz_opa
+  inbound_extauthz_timeout: 1s
+  inbound_extauthz_failuremodeallow: false
+```
+
+- Uses (as `opa-envoy-plugin`) a configmap to set OPA's policy. [This is not intended for production](https://github.com/open-policy-agent/opa-envoy-plugin#example-bundle-configuration). 
+- `opa-envoy-plugin` does not seem to react to changes on the configmap (is this intended?). To update policies, currently the OPA container needs to be restarted _(I believe I'm missing something here)_.
```

Comment:
Clarify ambiguity with a github issue?

Reply:
oh.... I need to clean up this comments, stale.